src/casp/services/api/data_manager/sql_templates/cell_analysis/match_metadata.sql.mako

```diff
@@ -0,0 +1,13 @@
+select
```

Comment:
Why is it called match metadata? Shouldn't be called `get_neighborhood_distance_summary` or something like that?